specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2020-07-01-preview/extensions.json

```diff
@@ -0,0 +1,736 @@
+      "enum": [
+        "Pending",
+        "Installed",
+        "Failed"
+      ],
+      "x-ms-enum": {
+        "name": "InstallStateType",
+        "modelAsString": true
+      }
+    },
+    "ExtensionStatus": {
+      "description": "Status from this instance of the extension.",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Status code provided by the Extension"
+        },
+        "displayStatus": {
+          "type": "string",
+          "description": "Short description of status of this instance of the extension."
+        },
+        "level": {
+          "type": "string",
+          "description": "Level of the status.",
+          "enum": [
+            "Error",
+            "Warning",
+            "Information"
+          ],
+          "default": "Information",
+          "x-ms-enum": {
+            "name": "LevelType",
+            "modelAsString": true
+          }
+        },
+        "message": {
+          "type": "string",
+          "description": "Detailed message of the status from the Extension instance."
+        },
+        "time": {
+          "type": "string",
+          "description": "DateLiteral (per ISO8601) noting the time of installation status."
+        }
+      }
+    },
+    "ConfigurationIdentity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal id of the system assigned identity which is used by the configuration."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant id of the system assigned identity which is used by the configuration."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of identity used for the configuration. Type 'SystemAssigned' will use an implicitly created identity. Type 'None' will not use Managed Identity for the configuration.",
+          "enum": [
+            "SystemAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceIdentityType",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "Identity for the managed cluster."
+    },
+    "ExtensionInstance": {
+      "description": "The Extension Instance object.",
+      "properties": {
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "description": "Properties of an Extension Instance resource",
+          "properties": {
+            "extensionType": {
+              "type": "string",
+              "description": "Type of the Extension, of which this resource is an instance of.  It must be one of the Extension Types registered with Microsoft.KubernetesConfiguration by the Extension publisher."
+            },
+            "autoUpgradeMinorVersion": {
```

Comment:
Is this property required to be specified in PUT? If not and if in your RP, a default value is assigned, then define the default value here.
Refer: https://armwiki.azurewebsites.net/rp_onboarding/process/property_design_best_practices.html#readwrite-properties-with-static-default-values

Reply:
Fixed